gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
```

Comment:
Sorry for the Log access, I disable the security now. You can see log and config [here](http://139.59.38.89:8080/job/xwfork/29/console) again.
I thought it was due to some mismatch of version, but it's fine if it is updated. 
These lib might help `sudo apt-get install lib32stdc++6 lib32z1 gcc-multilib`